rotkehlchen/externalapis/etherscan.py

```diff
@@ -627,7 +623,7 @@ def get_logs(
             module='logs',
             action='getLogs',
             options=options,
-            timeout=(DEFAULT_CONNECT_TIMEOUT, DEFAULT_READ_TIMEOUT * 2),
+            timeout=(CachedSettings().get_timeout_tuple()[0], CachedSettings().get_timeout_tuple()[1] * 2),  # noqa: E501
```

Comment:
you are asking for the same thing twice. Put it in a variable and reuse it in here.